test/java/src/com/ibm/streamsx/topology/test/api/ParallelTest.java

```diff
@@ -212,6 +212,167 @@ public void testAdjacentEndParallelUnionSource() throws Exception {
 	assertTrue(assertFinished.getResult().toString(), assertFinished.valid());
     }
 
+    @Test
+    public void testMultiAdjacentPartitionedParallelChildren() throws Exception {
+        checkUdpSupported();
+
+        List<String> stringList = getListOfUniqueStrings(800);
+        String stringArray[] = new String[800];
+        stringArray = stringList.toArray(stringArray);
+        Topology topology = newTopology("testAdj");
```

Comment:
Don't provide a name to `newTopology` and the name of the test will be used, helps in debugging tests.